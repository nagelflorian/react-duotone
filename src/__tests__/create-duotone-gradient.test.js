/* global describe, it, expect */
import createDuotoneGradient from '../create-duotone-gradient';

const primaryColor = [102, 250, 117];
const secondaryColor = [25, 4, 199];

describe('create-duotone-gradient', () => {
  it('should return the correct gradient map', () => {
    expect(
      createDuotoneGradient(primaryColor, secondaryColor),
    ).toMatchSnapshot();
  });
});
