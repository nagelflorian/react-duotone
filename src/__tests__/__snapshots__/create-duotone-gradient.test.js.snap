// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create-duotone-gradient should return the correct gradient map 1`] = `
Array [
  Array [
    25,
    4,
    199,
  ],
  Array [
    25,
    5,
    199,
  ],
  Array [
    26,
    6,
    198,
  ],
  Array [
    26,
    7,
    198,
  ],
  Array [
    26,
    8,
    198,
  ],
  Array [
    27,
    9,
    197,
  ],
  Array [
    27,
    10,
    197,
  ],
  Array [
    27,
    11,
    197,
  ],
  Array [
    27,
    12,
    196,
  ],
  Array [
    28,
    13,
    196,
  ],
  Array [
    28,
    14,
    196,
  ],
  Array [
    28,
    15,
    195,
  ],
  Array [
    29,
    16,
    195,
  ],
  Array [
    29,
    17,
    195,
  ],
  Array [
    29,
    18,
    194,
  ],
  Array [
    30,
    18,
    194,
  ],
  Array [
    30,
    19,
    194,
  ],
  Array [
    30,
    20,
    194,
  ],
  Array [
    30,
    21,
    193,
  ],
  Array [
    31,
    22,
    193,
  ],
  Array [
    31,
    23,
    193,
  ],
  Array [
    31,
    24,
    192,
  ],
  Array [
    32,
    25,
    192,
  ],
  Array [
    32,
    26,
    192,
  ],
  Array [
    32,
    27,
    191,
  ],
  Array [
    33,
    28,
    191,
  ],
  Array [
    33,
    29,
    191,
  ],
  Array [
    33,
    30,
    190,
  ],
  Array [
    33,
    31,
    190,
  ],
  Array [
    34,
    32,
    190,
  ],
  Array [
    34,
    33,
    189,
  ],
  Array [
    34,
    34,
    189,
  ],
  Array [
    35,
    35,
    189,
  ],
  Array [
    35,
    36,
    188,
  ],
  Array [
    35,
    37,
    188,
  ],
  Array [
    36,
    38,
    188,
  ],
  Array [
    36,
    39,
    187,
  ],
  Array [
    36,
    40,
    187,
  ],
  Array [
    36,
    41,
    187,
  ],
  Array [
    37,
    42,
    186,
  ],
  Array [
    37,
    43,
    186,
  ],
  Array [
    37,
    44,
    186,
  ],
  Array [
    38,
    45,
    185,
  ],
  Array [
    38,
    45,
    185,
  ],
  Array [
    38,
    46,
    185,
  ],
  Array [
    39,
    47,
    185,
  ],
  Array [
    39,
    48,
    184,
  ],
  Array [
    39,
    49,
    184,
  ],
  Array [
    39,
    50,
    184,
  ],
  Array [
    40,
    51,
    183,
  ],
  Array [
    40,
    52,
    183,
  ],
  Array [
    40,
    53,
    183,
  ],
  Array [
    41,
    54,
    182,
  ],
  Array [
    41,
    55,
    182,
  ],
  Array [
    41,
    56,
    182,
  ],
  Array [
    42,
    57,
    181,
  ],
  Array [
    42,
    58,
    181,
  ],
  Array [
    42,
    59,
    181,
  ],
  Array [
    43,
    60,
    180,
  ],
  Array [
    43,
    61,
    180,
  ],
  Array [
    43,
    62,
    180,
  ],
  Array [
    43,
    63,
    179,
  ],
  Array [
    44,
    64,
    179,
  ],
  Array [
    44,
    65,
    179,
  ],
  Array [
    44,
    66,
    178,
  ],
  Array [
    45,
    67,
    178,
  ],
  Array [
    45,
    68,
    178,
  ],
  Array [
    45,
    69,
    177,
  ],
  Array [
    46,
    70,
    177,
  ],
  Array [
    46,
    71,
    177,
  ],
  Array [
    46,
    72,
    176,
  ],
  Array [
    46,
    72,
    176,
  ],
  Array [
    47,
    73,
    176,
  ],
  Array [
    47,
    74,
    176,
  ],
  Array [
    47,
    75,
    175,
  ],
  Array [
    48,
    76,
    175,
  ],
  Array [
    48,
    77,
    175,
  ],
  Array [
    48,
    78,
    174,
  ],
  Array [
    49,
    79,
    174,
  ],
  Array [
    49,
    80,
    174,
  ],
  Array [
    49,
    81,
    173,
  ],
  Array [
    49,
    82,
    173,
  ],
  Array [
    50,
    83,
    173,
  ],
  Array [
    50,
    84,
    172,
  ],
  Array [
    50,
    85,
    172,
  ],
  Array [
    51,
    86,
    172,
  ],
  Array [
    51,
    87,
    171,
  ],
  Array [
    51,
    88,
    171,
  ],
  Array [
    52,
    89,
    171,
  ],
  Array [
    52,
    90,
    170,
  ],
  Array [
    52,
    91,
    170,
  ],
  Array [
    52,
    92,
    170,
  ],
  Array [
    53,
    93,
    169,
  ],
  Array [
    53,
    94,
    169,
  ],
  Array [
    53,
    95,
    169,
  ],
  Array [
    54,
    96,
    168,
  ],
  Array [
    54,
    97,
    168,
  ],
  Array [
    54,
    98,
    168,
  ],
  Array [
    55,
    99,
    167,
  ],
  Array [
    55,
    100,
    167,
  ],
  Array [
    55,
    100,
    167,
  ],
  Array [
    55,
    101,
    167,
  ],
  Array [
    56,
    102,
    166,
  ],
  Array [
    56,
    103,
    166,
  ],
  Array [
    56,
    104,
    166,
  ],
  Array [
    57,
    105,
    165,
  ],
  Array [
    57,
    106,
    165,
  ],
  Array [
    57,
    107,
    165,
  ],
  Array [
    58,
    108,
    164,
  ],
  Array [
    58,
    109,
    164,
  ],
  Array [
    58,
    110,
    164,
  ],
  Array [
    59,
    111,
    163,
  ],
  Array [
    59,
    112,
    163,
  ],
  Array [
    59,
    113,
    163,
  ],
  Array [
    59,
    114,
    162,
  ],
  Array [
    60,
    115,
    162,
  ],
  Array [
    60,
    116,
    162,
  ],
  Array [
    60,
    117,
    161,
  ],
  Array [
    61,
    118,
    161,
  ],
  Array [
    61,
    119,
    161,
  ],
  Array [
    61,
    120,
    160,
  ],
  Array [
    62,
    121,
    160,
  ],
  Array [
    62,
    122,
    160,
  ],
  Array [
    62,
    123,
    159,
  ],
  Array [
    62,
    124,
    159,
  ],
  Array [
    63,
    125,
    159,
  ],
  Array [
    63,
    126,
    158,
  ],
  Array [
    63,
    127,
    158,
  ],
  Array [
    64,
    127,
    158,
  ],
  Array [
    64,
    128,
    158,
  ],
  Array [
    64,
    129,
    157,
  ],
  Array [
    65,
    130,
    157,
  ],
  Array [
    65,
    131,
    157,
  ],
  Array [
    65,
    132,
    156,
  ],
  Array [
    65,
    133,
    156,
  ],
  Array [
    66,
    134,
    156,
  ],
  Array [
    66,
    135,
    155,
  ],
  Array [
    66,
    136,
    155,
  ],
  Array [
    67,
    137,
    155,
  ],
  Array [
    67,
    138,
    154,
  ],
  Array [
    67,
    139,
    154,
  ],
  Array [
    68,
    140,
    154,
  ],
  Array [
    68,
    141,
    153,
  ],
  Array [
    68,
    142,
    153,
  ],
  Array [
    68,
    143,
    153,
  ],
  Array [
    69,
    144,
    152,
  ],
  Array [
    69,
    145,
    152,
  ],
  Array [
    69,
    146,
    152,
  ],
  Array [
    70,
    147,
    151,
  ],
  Array [
    70,
    148,
    151,
  ],
  Array [
    70,
    149,
    151,
  ],
  Array [
    71,
    150,
    150,
  ],
  Array [
    71,
    151,
    150,
  ],
  Array [
    71,
    152,
    150,
  ],
  Array [
    72,
    153,
    149,
  ],
  Array [
    72,
    154,
    149,
  ],
  Array [
    72,
    154,
    149,
  ],
  Array [
    72,
    155,
    149,
  ],
  Array [
    73,
    156,
    148,
  ],
  Array [
    73,
    157,
    148,
  ],
  Array [
    73,
    158,
    148,
  ],
  Array [
    74,
    159,
    147,
  ],
  Array [
    74,
    160,
    147,
  ],
  Array [
    74,
    161,
    147,
  ],
  Array [
    75,
    162,
    146,
  ],
  Array [
    75,
    163,
    146,
  ],
  Array [
    75,
    164,
    146,
  ],
  Array [
    75,
    165,
    145,
  ],
  Array [
    76,
    166,
    145,
  ],
  Array [
    76,
    167,
    145,
  ],
  Array [
    76,
    168,
    144,
  ],
  Array [
    77,
    169,
    144,
  ],
  Array [
    77,
    170,
    144,
  ],
  Array [
    77,
    171,
    143,
  ],
  Array [
    78,
    172,
    143,
  ],
  Array [
    78,
    173,
    143,
  ],
  Array [
    78,
    174,
    142,
  ],
  Array [
    78,
    175,
    142,
  ],
  Array [
    79,
    176,
    142,
  ],
  Array [
    79,
    177,
    141,
  ],
  Array [
    79,
    178,
    141,
  ],
  Array [
    80,
    179,
    141,
  ],
  Array [
    80,
    180,
    140,
  ],
  Array [
    80,
    181,
    140,
  ],
  Array [
    81,
    182,
    140,
  ],
  Array [
    81,
    182,
    140,
  ],
  Array [
    81,
    183,
    139,
  ],
  Array [
    81,
    184,
    139,
  ],
  Array [
    82,
    185,
    139,
  ],
  Array [
    82,
    186,
    138,
  ],
  Array [
    82,
    187,
    138,
  ],
  Array [
    83,
    188,
    138,
  ],
  Array [
    83,
    189,
    137,
  ],
  Array [
    83,
    190,
    137,
  ],
  Array [
    84,
    191,
    137,
  ],
  Array [
    84,
    192,
    136,
  ],
  Array [
    84,
    193,
    136,
  ],
  Array [
    84,
    194,
    136,
  ],
  Array [
    85,
    195,
    135,
  ],
  Array [
    85,
    196,
    135,
  ],
  Array [
    85,
    197,
    135,
  ],
  Array [
    86,
    198,
    134,
  ],
  Array [
    86,
    199,
    134,
  ],
  Array [
    86,
    200,
    134,
  ],
  Array [
    87,
    201,
    133,
  ],
  Array [
    87,
    202,
    133,
  ],
  Array [
    87,
    203,
    133,
  ],
  Array [
    88,
    204,
    132,
  ],
  Array [
    88,
    205,
    132,
  ],
  Array [
    88,
    206,
    132,
  ],
  Array [
    88,
    207,
    131,
  ],
  Array [
    89,
    208,
    131,
  ],
  Array [
    89,
    209,
    131,
  ],
  Array [
    89,
    209,
    131,
  ],
  Array [
    90,
    210,
    130,
  ],
  Array [
    90,
    211,
    130,
  ],
  Array [
    90,
    212,
    130,
  ],
  Array [
    91,
    213,
    129,
  ],
  Array [
    91,
    214,
    129,
  ],
  Array [
    91,
    215,
    129,
  ],
  Array [
    91,
    216,
    128,
  ],
  Array [
    92,
    217,
    128,
  ],
  Array [
    92,
    218,
    128,
  ],
  Array [
    92,
    219,
    127,
  ],
  Array [
    93,
    220,
    127,
  ],
  Array [
    93,
    221,
    127,
  ],
  Array [
    93,
    222,
    126,
  ],
  Array [
    94,
    223,
    126,
  ],
  Array [
    94,
    224,
    126,
  ],
  Array [
    94,
    225,
    125,
  ],
  Array [
    94,
    226,
    125,
  ],
  Array [
    95,
    227,
    125,
  ],
  Array [
    95,
    228,
    124,
  ],
  Array [
    95,
    229,
    124,
  ],
  Array [
    96,
    230,
    124,
  ],
  Array [
    96,
    231,
    123,
  ],
  Array [
    96,
    232,
    123,
  ],
  Array [
    97,
    233,
    123,
  ],
  Array [
    97,
    234,
    122,
  ],
  Array [
    97,
    235,
    122,
  ],
  Array [
    97,
    236,
    122,
  ],
  Array [
    98,
    236,
    122,
  ],
  Array [
    98,
    237,
    121,
  ],
  Array [
    98,
    238,
    121,
  ],
  Array [
    99,
    239,
    121,
  ],
  Array [
    99,
    240,
    120,
  ],
  Array [
    99,
    241,
    120,
  ],
  Array [
    100,
    242,
    120,
  ],
  Array [
    100,
    243,
    119,
  ],
  Array [
    100,
    244,
    119,
  ],
  Array [
    100,
    245,
    119,
  ],
  Array [
    101,
    246,
    118,
  ],
  Array [
    101,
    247,
    118,
  ],
  Array [
    101,
    248,
    118,
  ],
  Array [
    102,
    249,
    117,
  ],
  Array [
    102,
    250,
    117,
  ],
]
`;
